plugins/woocommerce-blocks/assets/js/blocks/mini-cart/quantity-badge/index.tsx

```diff
@@ -36,6 +38,21 @@ const QuantityBadge = ( {
 		}
 	}
 
+	function determineCount( count: number, visibility: string ) {
```

Comment:
This feels quite complex, maybe we could do something like this:

`const shouldDisplayCount = productCountVisibility === 'always' || ( productCountVisibility === 'greater_than_zero' && count > 0 );`

instead?

Then hide the